tests/amountClaim/pages/submitted.js

```diff
@@ -29,5 +29,6 @@ module.exports = {
     I.see('1. Download the sealed claim form (this will open in a new window).')
     I.scrollPageToBottom()
     I.click('Download the sealed claim form')
+    I.wait(10)
```

Comment:
can't you do something better than this?
Wait till the request completes? Check for existence of the file maybe?
This adds 10 seconds on to every test that submits a claim